### ISSUE TITLE
disable progressbar in avtable_paged?

### ISSUE THREAD

**Issue by vjcitn** [Bioconductor/AnVIL#78]
```
seqtab = avtable_paged(table="sequencing")
## 
  |                                                                            
  |                                                                      |   0%
  |                                                                            
  |=                                                                     |   1%
  |                                                                            
  |=                                                                     |   2%
  |                                                                            
  |==                                                                    |   2%
  |                                                                            
  |==                                                                    |   3%
```
if used in vignette ... and this functionality may become more important ...

**Comment by vjcitn** [comment]
so in case the preceding is not completely clear it would be desirable to be able to suppress the progress bar output.  I can probably do it with message=FALSE in vignette but maybe closer to the call would be appropriate?

**Comment by mtmorgan** [comment]
Would it be ok to suppress specifically in non-interactive sessions (e.g., when knitting a vignette)?

**Comment by vjcitn** [comment]
no, message=FALSE is not sufficient.  results="hide"?

**Comment by vjcitn** [comment]
that sounds like a good idea, to suppress in non-interactive

**Comment by mtmorgan** [comment]
implemented in both bioc 3.17 and 3.18 (devel); AnVIL package 1.12.2 / 1.13.2
